### PR TITLE
Add link to es-kotlin-wrapper-client

### PR DESCRIPTION
**Pull request by jillesvangurp**
I recently created this client and think it is of interest to other users. I intend to continue adding features to it in the next months.

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
Yes, my details should be on file.
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
Yes
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by jillesvangurp** [comment]
I signed the agreement a few years ago and again just now. How do I move this forward?

**Comment by jillesvangurp** [comment]
ah now it went through. ignore my previous comment

File: docs/community-clients/index.asciidoc [hunk 1]
```diff
@@ -132,6 +132,8 @@ The following project appears to be abandoned:
 
 * https://github.com/mbuhot/eskotlin[ES Kotlin]:
   Elasticsearch Query DSL for kotlin based on the {client}/java-api/current/index.html[official Elasticsearch Java client].
+  
+* https://github.com/jillesvangurp/es-kotlin-wrapper-client[ES Kotlin Wrapper Client] Kotlin extension functions and abstractions for the {client}/java-api/current/index.html[official Elasticsearch Highlevel Client]. Aims to reduce the amount of boilerplate needed to do searches, bulk indexing and other common things users do with the client.
```
Comment (cbuescher) [inline docs/community-clients/index.asciidoc]: super small nit: could you add a colon after the link and put the decription on the next line? This seems to be the formatting in most other places on this page and improves readability of the source file IMHO. Also line wrapping
at about  ~120 chars would be nice although it isn't strictly necessary.

**Review by cbuescher — CHANGES_REQUESTED** [review]
Hi @jillesvangurp, looks great. Thanks for the PR and sorry for the delay here. I left a super small nitpicking comment in terms of formatting, other than that I'm happy to merge as soon as you have updated this.

**Comment by jillesvangurp** [comment]
@cbuescher done!

**Review by cbuescher — APPROVED** [review]
Great thanks, will merge now.

**Pull request merged**